src/lib/swapper/swappers/ThorchainSwapper/getThorTradeQuote/getTradeQuote.ts

```diff
@@ -114,13 +114,19 @@ export const getThorTradeQuote = async (
       slippageBps: thornodeQuote.slippage_bps,
       expectedAmountOutThorBaseUnit: thornodeQuote.expected_amount_out,
       isStreaming: false,
+      estimatedExecutionTimeMs: thornodeQuote.total_swap_seconds
+        ? 1000 * thornodeQuote.total_swap_seconds
+        : undefined,
```

Comment:
This will be pessimistic and return incorrect estimations , as we request both streaming swaps and regular swaps and `total_swap_seconds` is `total number of seconds a swap is expected to take (inbound conf + streaming swap + outbound delay)`, see with a bigly swap, it is the same as `streaming_swap_seconds`: https://paw.pt/h5uiNcax

For smaller amounts, this is also higher than `total_swap_seconds` which is definitely odd: https://paw.pt/h5ujGMNd

<img width="559" alt="image" src="https://github.com/shapeshift/web/assets/17035424/9695c6a3-492c-49f2-a5b8-126d3eb1a45e">

`streaming_swap_seconds` seems also inaccurate in and by itself, as it's not taking into account inbound confirmation + outbound delay: `approx the number of seconds the streaming swap will execute over`

After a bit of fiddling with requests, I reckon making a single request for both was theoretically fine previously but won't work for getting the time estimate right, and we should make two requests, one with `streaming_interval` and one without, and use the `total_swap_seconds` from the relevant one, see

- With streaming interval: https://paw.pt/h5umIqcQ
- Without streaming interval: https://paw.pt/h5umNrgG

Reply:
🐐

Reply:
Did you pull the latest commit @gomesalexandre? I was getting that before this guy: https://github.com/shapeshift/web/pull/5118/commits/1b015a45389b6d7a10038db8cf3cdd6121c30166

Reply:
Indeed, confirmed fixed in https://github.com/shapeshift/web/commit/1b015a45389b6d7a10038db8cf3cdd6121c30166!